pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -742,6 +742,12 @@ private void subscribeTopicPartitions(CompletableFuture<Void> subscribeResult, S
 
         List<CompletableFuture<Consumer<T>>> futureList;
 
+        try {
+            client.preProcessSchemaBeforeSubscribe(client, schema, topicName);
+        } catch (Throwable t) {
+            subscribeResult.completeExceptionally(t);
```

Comment:
@congbobo184 don't you need to return here?

Reply:
yes, It should be